src/chartmuseum/Chart.yaml

```diff
@@ -1,11 +1,18 @@
-apiVersion: v1
+apiVersion: v2
 description: Host your own Helm Chart Repository
 name: chartmuseum
-version: 2.16.0
+version: 3.0.0
 appVersion: 0.13.0
-home: https://github.com/helm/chartmuseum
-icon: https://raw.githubusercontent.com/chartmuseum/charts/main/logo.jpg 
+home: https://github.com/chartmuseum
```

Comment:
Can you please change the homepage back to the main repo?

Reply:
Sure. done